Formalisation(shacl)/Core/PiecesShape/Catalog.ttl

```diff
@@ -35,7 +35,7 @@
    sh:minCount 0 ;
   ],
   [
-    sh:path <http://www.w3.org/ns/dcat#DataService> ;
+    sh:path <http://www.w3.org/ns/dcat#Service> ;
```

Comment:
Predicates are by convention start with a lowercase character; this should be `dcat:service` as per [the spec](https://www.w3.org/TR/vocab-dcat-3/#Property:catalog_service).